test/test.js

```diff
@@ -0,0 +1,126 @@
+var assert = require('assert');
+const t = require('../index');
+const {
+    FlightPriceWatcher
+} = require('../index');
+
+// /!\ Go to the botton and add the API KEY before testing /!\
+
+describe('FlightPriceWatcher', () => {
+    describe('#constructor()', function () {
+        it('should construct', function () {
+            let app = new FlightPriceWatcher();
+            assert.ok(app);
+        });
+    });
+    describe('#run()', () => {
+        // Check if the keyboard key is the right color when none is stored
+        // Means getLastPrice returns null
+        it('Can get price when none is stored', function () {
+            let app = new FlightPriceWatcher();
+            assert.ok(app.setLastPrice(null));
+            assert.equal(app.getLastPrice(), null);
+            // Check if the date format is correct
+            assert.ok(app.formatDate());
```

Comment:
Should create separate test for this method:
- one with a correct format
- one with a wrong format